bin/run-ansible.sh

```diff
@@ -156,13 +160,21 @@ if [[ "${playbook}" = "server_and_nuc_deploy" ]]; then
     echo " ---------------------------------------------------"
     echo ""
 elif [[ "${playbook}" = "teleop_deploy" ]]; then
-    ansible_flags="${ansible_flags} --ask-become-pass --ask-pass "
-    aurora_inventory="ansible/inventory/teleop/${aurora_inventory}"
+    ansible_flags="${ansible_flags} --ask-vault-pass"
+    if [[ "${aurora_inventory}" = "" ]]; then
+        if [[ $extra_vars == *"remote_teleop=true"* ]]; then
+            aurora_inventory="ansible/inventory/teleop/production_remote"
+        else
+            aurora_inventory="ansible/inventory/teleop/production"
+        fi
+    else
+        aurora_inventory="ansible/inventory/teleop/${aurora_inventory}"
+    fi
+
     echo ""
     echo " ---------------------------------------------------"
-    echo " |             SSH and BECOME passwords:           |"
-    echo " | SSH: enter the NUC sudo password                |"
-    echo " | BECOME: just press enter (same as SSH password) |"
+    echo " |                 VAULT password:                 |"
+    echo " | Enter the VAULT password provided by Shadow     |"
```

Comment:
so no need to have ssh password anymore? What if I want to use it on my setup here in Bristol?

Reply:
change the NUC password to the same password as delivery NUCs (I will tell you in chat)